README.md

```diff
@@ -1,3 +1,517 @@
 # React Native DateTimePicker
 
-[![CircleCI](https://circleci.com/gh/react-native-community/react-native-datetimepicker.svg?style=svg)](https://circleci.com/gh/react-native-community/react-native-datetimepicker)
+[![CircleCI Status](https://img.shields.io/circleci/project/github/react-native-community/react-native-datetimepicker/master.svg)](https://circleci.com/gh/react-native-community/workflows/react-native-datetimepicker/tree/master) ![Supports Android and iOS](https://img.shields.io/badge/platforms-android%20|%20ios-lightgrey.svg) ![MIT License](https://img.shields.io/npm/l/@react-native-community/slider.svg)
+
+React Native date & time picker component for iOS and Android
+
+## Getting started
+
+`npm install react-native-datetimepicker --save`
+
+or
+
+`yarn add react-native-datetimepicker`
+
+
+### Using React Native Link
+
+`react-native link react-native-datetimepicker`
+
+### Manual installation
+
+#### iOS
+
+- Linking project manually:
+  
+  1. In the XCode's "Project navigator", right click on your project's Libraries folder ➜ `Add Files to <...>`.
+  2. Go to `node_modules` ➜ `react-native-datetimepicker` ➜ `ios` ➜ select `RNDateTimePicker.xcodeproj`.
+  3. Add `libRNDateTimePicker.a` to `Build Phases -> Link Binary With Libraries`.
+
+- Using CocoaPods:
+
+  1. Install CocoaPods, here the [installation guide](https://guides.cocoapods.org/using/getting-started.html).
+  2. Inside the iOS folder run `pod init`, this will create the initial `pod` file.
```

Comment:
This section is ok to have, but it feels like the focus should be more on adding `RNDateTimePicker` from the podspec. Right now it is focused on setting up the consuming app, which most avid cocoapods users might have already done when they initialized their own project.